lib/matplotlib/figure.py

```diff
@@ -2657,10 +2657,9 @@ def set_size_inches(self, w, h=None, forward=True):
         if forward:
             canvas = getattr(self, 'canvas')
             if canvas is not None:
-                dpi_ratio = getattr(canvas, '_dpi_ratio', 1)
```

Comment:
Note, this always returned 1 since the attribute was renamed, so the resize was always in physical pixels.